src/vector_index.c

```diff
@@ -447,3 +493,12 @@ VecSimResolveCode VecSim_ResolveQueryParams(VecSimIndex *index, VecSimRawParam *
   QueryError_SetErrorFmt(status, RSErrorCode, "Error parsing vector similarity parameters: %s", error_msg);
   return vecSimCode;
 }
+
+void VecSim_TieredParams_Init(TieredIndexParams *params, StrongRef sp_ref) {
+  params->primaryIndexParams = rm_calloc(1, sizeof(VecSimParams));
+#ifdef POWER_TO_THE_WORKERS
+  params->jobQueue = _workers_thpool;
+#endif
```

Comment:
We expect that `_workers_thpool` is NULL if  `POWER_TO_THE_WORKERS is false, and that  `params->jobQueue` is initialized with NULL .